manager/integration/negative_tests/replica_resiliency_test.robot

```diff
@@ -0,0 +1,53 @@
+*** Settings ***
+Documentation    Negative Test Cases
+
+Library    ../framework/steps/volume_steps.py
+
+Test setup    set_test_name    ${TEST NAME}
+
+Test Teardown    cleanup_resources
+
+*** Variables ***
+${LOOP_COUNT}    20
+
+*** Test Cases ***
+Replica Rebuilding While Replica Deletion
+    Create a volume 2 GB with 3 replicas
+    Write data to the volume
+    FOR    ${i}    IN RANGE    ${LOOP_COUNT}
+        Delete replica 0 to trigger replica 0 rebuilding
+        During replica 0 rebuilding, delete replica 1
+        Wait until replica 0 rebuilt, delete replica 2
+        Check data is intact
+        Wait until all replicas rebuilt
+    END
+
+*** Keywords ***
```

Comment:
These keywords can be placed in different keyword files, so they can be reused in other test cases later. 

How to decide what keyword files should be created. You can base on types of Longhorn resources. For example, volume related, you can add those keywords to `keywords/volume.robot`. Replica related, can have `replica.robot`.

About folder structure, it can be:
- libs (python functions for programmable keywords)
- tests (robot test cases)
- keywords (robot keywords)

Reply:
refined.